contents/handbook/people/team-structure/platform/objectives.mdx

```diff
@@ -1,11 +1,30 @@
-* **Proposed Objective**:
-    * PostHog Cloud in the US remains reliable and launch EU Cloud
-* **Proposed Key Results**:
-    * Multiple organizations using EU hosted PostHog Cloud without regressions in either EU or US cloud
-    * Reliable monitoring and alerting on all key services
-    * Upgrade to Kubernetes 1.25 to unblock key customers
-    * Meet SOC 2 compliance requirements for all cloud infrastructure
-* **Rationale**:
-    * To nail self-serve we need to make it as easy as possible for customers to sign up and get started
-    * For businesses in the EU today, the main barrier is needing to self host to keep data in the EU, this limits trust in our brand and the potential adoption we can drive
-    * Shipping a secure cloud in the EU will also encourage many businesses to get started with PostHog Cloud, even if they ultimately intend to self-host
\ No newline at end of file
+- Objective: High uptime and increase the resilience/redundancy
+    - Key results:
```

Comment:
Is it possible to shift focus to the measurable outputs of this work, rather than listing the KRs as yes/no checkboxes? For example -- what will a migration of postgres from Heroku achieve? Cost savings, an increase in speed, etc? What will regular load testing accomplish?